test/nodes/conftest.py

```
@@ -0,0 +1,516 @@
+sys.modules['rocketlib.types'] = _mock_rocketlib_types
+
+
+# ---------------------------------------------------------------------------
+# 3. Mock: ai.common tree
+# ---------------------------------------------------------------------------
+
+
+class MockQuestion:
+    """Simulates ai.common.schema.Question."""
+
+    def __init__(self, text='test question'):
```

Ruff (0.15.7)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)